homeassistant/components/conversation/agent.py

```diff
@@ -48,8 +48,10 @@ def supported_languages(self) -> list[str] | Literal["*"]:
     async def async_process(self, user_input: ConversationInput) -> ConversationResult:
         """Process a sentence."""
 
+    @abstractmethod
     async def async_reload(self, language: str | None = None) -> None:
         """Clear cached intents for a language."""
 
+    @abstractmethod
     async def async_prepare(self, language: str | None = None) -> None:
         """Load intents for a language."""
```

Comment:
Okay, these two functions are only implemented by DefaultAgent, but no other of the Agent classes. I will revert this rule for now because fixing this would need more work.